testsuite/scripts/pom.xml

```diff
@@ -80,6 +109,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-maven-plugin</artifactId>
+                <version>${version.wildfly.plugin}</version>
+                <configuration>
+                    <skip>${skipTests}</skip>
+                    <jboss-home>${wildfly.home}</jboss-home>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>configure-standalone</id>
+                        <phase>process-test-classes</phase>
+                        <goals>
+                            <goal>start</goal>
+                            <goal>execute-commands</goal>
+                            <goal>shutdown</goal>
```

Comment:
Ideally this should be done in an embedded server. However the one issue is the `logging.properties` needs to be updated. The main issue with running `start` first is if the `execute-commands` fails it leaves a server running.